components/Loading.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "styled-components/native";
+import LottieView from "lottie-react-native";
```

Comment:
The loading Lottie is really pretty! It matches the design quite well.

Reply:
😍